content/en/docs/instrumentation/net/manual.md

```diff
@@ -0,0 +1,253 @@
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+// Define some important constants and the activity source
+var serviceName = "MyCompany.MyProduct.MyService";
+var serviceVersion = "1.0.0";
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Configure important OpenTelemetry settings, the console exporter, and automatic instrumentation
+builder.Services.AddOpenTelemetryTracing(b =>
+{
+    b
+    .AddConsoleExporter()
+    .AddSource(serviceName)
+    .SetResourceBuilder(
+        ResourceBuilder.CreateDefault()
+            .AddService(serviceName: serviceName, serviceVersion: serviceVersion))
+});
+```
+
+This is also where you can configure instrumentation libraries.
+
+Note that this sample uses the Console Exporter. If you are exporting to another endpoint,
+you'll have to use a different exporter.
+
+## Setting up an ActivitySource
+
+Once tracing is initialized, you can configure an `ActivitySource`, which will be how
+you trace operations with `Activity`s.
+
+Typically, an `ActivitySource` is instantiated once per app/service that is being instrumented,
+so it's a good idea to instantiate it once in a shared location. It is also typically named
+the same as the Service Name.
+
+```csharp
+using System.Diagnostics;
+
+public static class Telemetry
+{
+    //...
+
+    // 'TelemetryConstants.ServiceName' comes from somewhere in your code
+    public static readonly ActivitySource MyActivitySource = new(TelemetryConstants.ServiceName)
+
+    //...
+}
+```
+
+You can instantiate several `ActivitySource`s if that suits your scenario.
+
+## Creating Activities
+
+To create an activity, give it a name and create it from your `ActivitySource`.
+
+```csharp
+using var myActivity = MyActivitySource.StartActivity("SayHello")
+
+// do work that 'myActivity' will now track
+```
+
+## Creating nested Activities
+
+If you have a distinct sub-operation you'd like to track as a part of another one,
+you can create activities to represent the relationship.
+
+```csharp
+public static void ParentOperation()
+{
+    using var parentOperation = MyActivitySource.StartActivity("Parent")
+
+    // Do some work in ParentOperation
+
+    ChildOperation();
+
+    // Finish up some work in ParentOperation
+}
+
+public static void ChildOperation()
+{
+    using var childOperation = MyActivitySource.StartActivity("Child")
+
+    // Do the child operation
+}
+```
+
+When you view spans in a trace visualization tool, `Child` will be tracked as a nested
+operation under `Parent`.
+
+## Get the current Activity
+
+Sometimes it's helpful to access whatever the current `Activity` is at a point in time so you can enrich
+it with more information.
+
+```csharp
+var activity = Activity.Current;
+// may be null if there is none
+```
+
+Note that `using` is not used in the following example. Doing so will end current `Activity`,
+which is not likely to be desired.
+
+## Add tags to an Activity
```

Comment:
I'd go with 'attributes' since that's the OTel term, then clarify that .NET Activity Tags === OTel Attributes.

Reply:
I went with tags since every other subsection uses the System.Diagnostics terminology. The other idea I had was to use OTel terminology, but in each section describe the System.Diagnostics-equivalent.

I want to also create a "manual instrumentation but with the shim" doc too (named better) that would do this

Reply:
Happy to change btw, just wanted to explain the rationale

Reply:
I think it'd be good to at least have a callout for 'hey, this is the same thing as an attribute' so we can be consistent across languages.

Reply:
Doneski